sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -874,6 +875,69 @@ class AdaptiveQueryExecSuite
     }
   }
 
+  test("SPARK-44426: Optimize adaptive skew join for ExistenceJoin") {
+    withSQLConf(
+      SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> "-1",
+      SQLConf.COALESCE_PARTITIONS_MIN_PARTITION_NUM.key -> "1",
+      SQLConf.SHUFFLE_PARTITIONS.key -> "100",
+      SQLConf.SKEW_JOIN_SKEWED_PARTITION_THRESHOLD.key -> "800",
+      SQLConf.ADVISORY_PARTITION_SIZE_IN_BYTES.key -> "800") {
+      withTempView("skewed", "nonSkewed") {
+        // skewData1
+        spark
+          .range(0, 1000, 1, 10)
+          .select(
+            when('id < 250, 249)
```

Comment:
please do not user `'` to represent the symbol which is deprecated. You can use `$"id"` instead